_source/_posts/2021-11-12-native-sso.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: blog_post
+title: "Native SSO: Desktop and Mobile Apps Single Sign-On"
+author: [huan-liu, daniel-zeller]
+by: internal-contributor
+communities: [mobile]
+description: "Tutorial: Learn how to enable Single Sign-On for mobile and desktop applications."
+tags: [ios, macos, icloud, keychain, oauth, security, sso, native-sso, mobile, desktop]
+tweets:
+- "Simplify authentication between mobile applications with Native single sign-on 🔒"
+- "Single sign-on isn't just for web applications. Learn about the new #OIDC spec for 'native SSO' 🛡️"
+- "Did you know you can share authenticated sessions between iOS/macOS applications!🍎"
+image:
+type: conversion
+---
+
+When you hear SSO (Single Sign-On), you probably immediately think of web apps, and how you only need to sign into one web app first, then all other web apps give you seamless access. Unfortunately, for desktop and mobile applications (often referred to as "native apps"), the seamless web SSO experience (also referred to as native SSO) has not caught on. This is partly due to the lack of an industry standard or best practice on how to accomplish native SSO.
+
+Although there is no standard yet, there's been growing demand for a solution, due to the continued expansion of native apps, both desktop and mobile. In 2020, a report from RiskIQ counted [8.9 million mobile apps](https://www.forbes.com/sites/johnkoetsier/2020/02/28/there-are-now-89-million-mobile-apps-and-china-is-40-of-mobile-app-spending/?sh=33b7e96721dd). There are also a large number of internal native apps available to employees within an enterprise.
+
+In comparison with desktop web apps, it's even more critical to enable SSO for mobile apps, due to the smaller keyboard and screen on a mobile device, which makes password input more painful. To date, complex proprietary solutions have not proven interoperable with the variety of native applications an enterprise installs on employee devices. Thus, there is still a dire need for an industry-standard solution.
```

Comment:
```suggestion
In comparison with desktop or web apps, it's even more critical to enable SSO for mobile apps, due to the smaller keyboard and screen on a mobile device, which makes password input more painful. To date, complex proprietary solutions have not proven interoperable with the variety of native applications an enterprise installs on employee devices. Thus, there is still a dire need for an industry-standard solution.
```

@huanliu-okta can you confirm I'm not 100% sure what this should be

Reply:
Confirm. Good catch!